src/makernote_int.cpp

```diff
@@ -276,11 +274,9 @@ namespace Exiv2 {
         return sizeOfSignature();
     } // OlympusMnHeader::write
 
-    const byte Olympus2MnHeader::signature_[] = {
```

Comment:
How about converting this to an `std::array`, then the `sizeOfSignature` function is no longer required.

Reply:
We would need to count then. It all is compile-teime evaluated now -> no more overhead, which is the reason why coverage increased (since it actually is uncovered).